src/routes/Ecosystem/MellowLPTable/components/MellowLPEntry/MellowLPEntry.styled.tsx

```diff
@@ -2,49 +2,56 @@ import Box from '@mui/material/Box';
 import { styled } from '@mui/material/styles';
 
 import { Typography } from '../../../../../components/atomic/Typography/Typography';
-import { Tag } from '../Tag/Tag';
 
 export const MellowLPEntryContainerBox = styled(Box)`
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
+  align-items: center;
+  padding: 0px;
 `;
 
 export const MellowLPEntryInfoBox = styled(Box)`
+  box-sizing: border-box;
+
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
   padding: ${({ theme }) => theme.spacing(4)};
-  background-color: #19152a;
-  border-radius: 8px 8px 0 0;
+  gap: ${({ theme }) => theme.spacing(6)};
+
+  background: #19152a;
+  border-width: 1px 1px 0px 1px;
+  border-style: solid;
+  border-color: #1e1933;
```

Comment:
Use a color from constants. Is it possible?

Reply:
https://github.com/Voltz-Protocol/voltz-ui/commit/d6cedb705445c150c0de0460c9ff447f5ea867d3